src/cli/plugin/setting_parser.js

```diff
@@ -1,5 +1,6 @@
 const { resolve } = require('path');
 const expiry = require('expiry-js');
+import intersection from 'lodash';
```

Comment:
This should actually be `import { intersection } from 'lodash';` The current line should error.